Result.podspec

```diff
@@ -14,6 +14,6 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target = '9.0'
   
-  s.swift_version = '4.0'
-  s.cocoapods_version = '>= 1.4.0'
+  s.swift_version = '4.1'
+  s.cocoapods_version = '>= 1.5.0'
```

Comment:
Why is CocoaPods 1.5.0 required? 1.4.0 requirement is to use `s.swift_version` syntax. Is there addional requirment?